CHANGELOG.md

```diff
@@ -1,4 +1,13 @@
 # CHANGELOG
+## 0.0.36
+- [common] Improve MinecraftVersion added known version (1.8 to 1.21.1) with protocol number and some helper methods
+  - `MinecraftVersion#getVersion`
```

Comment:
We don't need to get into so much details here, its better to keep only the first line